ElectronicObserver/Data/KCDatabase.cs

```diff
@@ -178,6 +179,11 @@ public sealed class KCDatabase : IKCDatabase
 	private SystemQuestTrackerManager? _systemQuestTrackerManager;
 	public SystemQuestTrackerManager SystemQuestTrackerManager => _systemQuestTrackerManager ??= new();
 
+
+	private EquipmentUpgradePlanManager? _equipmentUpgradePlanManager;
+	public EquipmentUpgradePlanManager EquipmentUpgradePlanManager => _equipmentUpgradePlanManager ??= new();
```

Comment:
Is having the planner here needed? For quest trackers for example it was needed, so it's available in fleet view, but I think the upgrade planner won't be needed outside its UI.

Reply:
I'm also making an equipment plan view so it should be needed i think

Reply:
I see, I didn't consider making a view for it, but yeah, in that case ignore this comment.